<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Model</title>
    <!-- CSS -->
    <link href="../asserts/css/bootstrap.min.css" rel="stylesheet">
    <link href="../asserts/css/style.css" rel="stylesheet">


</head>

<body>
    <!-- loading -->
    <div class="loading-body">
        <!-- 绘制齿轮 -->
        <div class="load">
            <div class="gear first">
                <svg id="purple" viewbox="0 0 100 100" fill="#afb4db">
                    <path
                        d="m88.22905,52.73842l12.66081,-6.69624c-0.24889,-2.48103 -0.7299,-4.88779 -1.33165,-7.25374l-14.06435,-1.94757c-1.21732,-2.93246 -2.82483,-5.65223 -4.71443,-8.14886l5.36114,-13.18525c-1.76284,-1.71922 -3.66664,-3.28664 -5.67042,-4.72891l-11.94246,7.49732c-2.80052,-1.47273 -5.81781,-2.59129 -8.98524,-3.30968l-4.31582,-13.37042c-1.22401,-0.09243 -2.44766,-0.19008 -3.69591,-0.19008s-2.46845,0.0954 -3.69613,0.19008l-4.27306,13.25199c-3.25234,0.69537 -6.33625,1.82692 -9.20876,3.31783l-11.75647,-7.38703c-2.00124,1.44227 -3.90496,3.0097 -5.66766,4.72891l5.2136,12.82104c-2.03201,2.60429 -3.73326,5.47776 -5.01105,8.57467l-13.62035,1.8837c-0.59921,2.36263 -1.08274,4.76684 -1.32906,7.25345l12.2246,6.46784c0.1385,3.4047 0.70298,6.7044 1.70254,9.8117l-9.22158,10.19626c1.07271,2.22493 2.26701,4.38264 3.64861,6.40699l13.5767,-2.91464c2.226,2.422 4.76215,4.54406 7.56029,6.29894l-0.51461,13.89098c2.22975,1.00564 4.54649,1.84729 6.93396,2.52003l8.5892,-11.04055c1.52399,0.19456 3.06723,0.3256 4.64128,0.3256c1.70519,0 3.37278,-0.15405 5.01352,-0.37947l8.63392,11.1018c2.39122,-0.67196 4.70444,-1.51356 6.93395,-2.51926l-0.52454,-14.14044c2.66699,-1.72661 5.08534,-3.79226 7.21739,-6.12444l13.92978,2.98893c1.38161,-2.02403 2.57607,-4.17945 3.64839,-6.40624l-9.52026,-10.5301c0.91453,-2.94323 1.42539,-6.05022 1.57412,-9.25513l0,0l0.00001,-0.00001zm-21.0432,16.08496l-6.00271,3.90775l-3.69207,-5.67259c-1.92067,0.73877 -3.98849,1.18799 -6.1683,1.18799c-9.52022,0 -17.23546,-7.71828 -17.23546,-17.23701c0,-9.51912 7.71525,-17.23441 17.23546,-17.23441c9.51773,0 17.2358,7.71529 17.2358,17.23441c0,4.75679 -1.92697,9.05688 -5.04179,12.17683l3.66907,5.63703z">
                    </path>
                </svg>
            </div>
            <div class="gear second">
                <svg id="pink" viewbox="0 0 100 100" fill="#FFB5C5">
                    <path
                        d="m88.22905,52.73842l12.66081,-6.69624c-0.24889,-2.48103 -0.7299,-4.88779 -1.33165,-7.25374l-14.06435,-1.94757c-1.21732,-2.93246 -2.82483,-5.65223 -4.71443,-8.14886l5.36114,-13.18525c-1.76284,-1.71922 -3.66664,-3.28664 -5.67042,-4.72891l-11.94246,7.49732c-2.80052,-1.47273 -5.81781,-2.59129 -8.98524,-3.30968l-4.31582,-13.37042c-1.22401,-0.09243 -2.44766,-0.19008 -3.69591,-0.19008s-2.46845,0.0954 -3.69613,0.19008l-4.27306,13.25199c-3.25234,0.69537 -6.33625,1.82692 -9.20876,3.31783l-11.75647,-7.38703c-2.00124,1.44227 -3.90496,3.0097 -5.66766,4.72891l5.2136,12.82104c-2.03201,2.60429 -3.73326,5.47776 -5.01105,8.57467l-13.62035,1.8837c-0.59921,2.36263 -1.08274,4.76684 -1.32906,7.25345l12.2246,6.46784c0.1385,3.4047 0.70298,6.7044 1.70254,9.8117l-9.22158,10.19626c1.07271,2.22493 2.26701,4.38264 3.64861,6.40699l13.5767,-2.91464c2.226,2.422 4.76215,4.54406 7.56029,6.29894l-0.51461,13.89098c2.22975,1.00564 4.54649,1.84729 6.93396,2.52003l8.5892,-11.04055c1.52399,0.19456 3.06723,0.3256 4.64128,0.3256c1.70519,0 3.37278,-0.15405 5.01352,-0.37947l8.63392,11.1018c2.39122,-0.67196 4.70444,-1.51356 6.93395,-2.51926l-0.52454,-14.14044c2.66699,-1.72661 5.08534,-3.79226 7.21739,-6.12444l13.92978,2.98893c1.38161,-2.02403 2.57607,-4.17945 3.64839,-6.40624l-9.52026,-10.5301c0.91453,-2.94323 1.42539,-6.05022 1.57412,-9.25513l0,0l0.00001,-0.00001zm-21.0432,16.08496l-6.00271,3.90775l-3.69207,-5.67259c-1.92067,0.73877 -3.98849,1.18799 -6.1683,1.18799c-9.52022,0 -17.23546,-7.71828 -17.23546,-17.23701c0,-9.51912 7.71525,-17.23441 17.23546,-17.23441c9.51773,0 17.2358,7.71529 17.2358,17.23441c0,4.75679 -1.92697,9.05688 -5.04179,12.17683l3.66907,5.63703z">
                    </path>
                </svg>
            </div>
            <div class="gear third">
                <svg id="orange" viewbox="0 0 100 100" fill="#fedcbd">
                    <path
                        d="m88.22905,52.73842l12.66081,-6.69624c-0.24889,-2.48103 -0.7299,-4.88779 -1.33165,-7.25374l-14.06435,-1.94757c-1.21732,-2.93246 -2.82483,-5.65223 -4.71443,-8.14886l5.36114,-13.18525c-1.76284,-1.71922 -3.66664,-3.28664 -5.67042,-4.72891l-11.94246,7.49732c-2.80052,-1.47273 -5.81781,-2.59129 -8.98524,-3.30968l-4.31582,-13.37042c-1.22401,-0.09243 -2.44766,-0.19008 -3.69591,-0.19008s-2.46845,0.0954 -3.69613,0.19008l-4.27306,13.25199c-3.25234,0.69537 -6.33625,1.82692 -9.20876,3.31783l-11.75647,-7.38703c-2.00124,1.44227 -3.90496,3.0097 -5.66766,4.72891l5.2136,12.82104c-2.03201,2.60429 -3.73326,5.47776 -5.01105,8.57467l-13.62035,1.8837c-0.59921,2.36263 -1.08274,4.76684 -1.32906,7.25345l12.2246,6.46784c0.1385,3.4047 0.70298,6.7044 1.70254,9.8117l-9.22158,10.19626c1.07271,2.22493 2.26701,4.38264 3.64861,6.40699l13.5767,-2.91464c2.226,2.422 4.76215,4.54406 7.56029,6.29894l-0.51461,13.89098c2.22975,1.00564 4.54649,1.84729 6.93396,2.52003l8.5892,-11.04055c1.52399,0.19456 3.06723,0.3256 4.64128,0.3256c1.70519,0 3.37278,-0.15405 5.01352,-0.37947l8.63392,11.1018c2.39122,-0.67196 4.70444,-1.51356 6.93395,-2.51926l-0.52454,-14.14044c2.66699,-1.72661 5.08534,-3.79226 7.21739,-6.12444l13.92978,2.98893c1.38161,-2.02403 2.57607,-4.17945 3.64839,-6.40624l-9.52026,-10.5301c0.91453,-2.94323 1.42539,-6.05022 1.57412,-9.25513l0,0l0.00001,-0.00001zm-21.0432,16.08496l-6.00271,3.90775l-3.69207,-5.67259c-1.92067,0.73877 -3.98849,1.18799 -6.1683,1.18799c-9.52022,0 -17.23546,-7.71828 -17.23546,-17.23701c0,-9.51912 7.71525,-17.23441 17.23546,-17.23441c9.51773,0 17.2358,7.71529 17.2358,17.23441c0,4.75679 -1.92697,9.05688 -5.04179,12.17683l3.66907,5.63703z">
                    </path>
                </svg>
            </div>

            <!-- 绘制圆形水珠 -->
            <div class="lil-circle"></div>
            <svg class="blur-circle">
                <filter id="blur">
                    <fegaussianblur in="SourceGraphic" stddeviation="13"></fegaussianblur>
                </filter>
            </svg>
        </div>
        <div class="loading-text">Loading · · ·</div>
    </div>
    <!-- navbar -->
    <nav class="navbar navbar-expand-lg navbar-scrolled justify-content-between">
        <a class="navbar-brand" href="#">
            <img src="../asserts/img/logo-light.svg" alt="Logo" class="logo dark-logo">
            <img src="../asserts/img/logo-dark.svg" alt="Logo" class="logo light-logo ">
        </a>

        <!-- 汉堡按钮 -->
        <button class="navbar-toggler p-0 border-0" type="button" data-toggle="collapse" data-target="#navbarNav"
            aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <?xml version="1.0" standalone="no"?>
            <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg
                t="1695294825594" class="icon menu-icon" viewBox="0 0 1024 1024" version="1.1"
                xmlns="http://www.w3.org/2000/svg" p-id="4266" xmlns:xlink="http://www.w3.org/1999/xlink" width="200"
                height="200">
                <path
                    d="M815 192.7H207.3c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16H815c8.8 0 16-7.2 16-16v-32c0-8.9-7.1-16-16-16zM815 480.8H463.8c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16H815c8.8 0 16-7.2 16-16v-32c0-8.9-7.1-16-16-16zM815 767.9H207.3c-8.8 0-16 7.2-16 16v32c0 8.8 7.2 16 16 16H815c8.8 0 16-7.2 16-16v-32c0-8.8-7.1-16-16-16zM303.8 618.1c14.4 12.2 36.5 2 36.5-16.9V421.1c0-18.9-22.2-29.1-36.5-16.8l-101.1 86.4c-11.5 11.5-11.5 30.2 0.1 41.7l101 85.7z"
                    p-id="4267"></path>
            </svg>
        </button>



        <div class="collapse navbar-collapse justify-content-center nav-right" id="navbarNav">
            <ul class="navbar-nav d-flex align-items-center">
                <li class="nav-item dropdown ">
                    <a class="nav-link en" href="../index.html" i18n="home">Home</a>
                    <a class="nav-link d-none cn" href="../index.html" i18n="home">主页</a>
                </li>
                <li class="nav-item dropdown">
                    <a class="nav-link en" href="#">Project</a>
                    <a class="nav-link d-none cn" href="#">项目</a>
                    <div class="dropdown-menu">
                        <a class="dropdown-item en" href="./description.html">Desciption</a>
                        <a class="dropdown-item d-none cn" href="./description.html">描述</a>
                        <a class="dropdown-item en" href="./design.html">Design</a>
                        <a class="dropdown-item d-none cn" href="./design.html">设计</a>
                        <a class="dropdown-item en" href="./model.html">Model</a>
                        <a class="dropdown-item d-none cn" href="./model.html">模型</a>
                        <!-- <a class="dropdown-item en" href="./hardware.html" i18n="hardware">Hardware</a> -->
                        <!-- <a class="dropdown-item d-none cn" href="./hardware.html" i18n="hardware">硬件</a> -->

                        <a class="dropdown-item en" href="./reports.html">Reports</a>
                        <a class="dropdown-item d-none cn" href="./reports.html" ">报告</a>
                        <a class=" dropdown-item en" href="./Supplenmentary_information.html">Supplenmentary
                            information</a>
                        <a class="dropdown-item d-none cn" href="./Supplenmentary_information.html">支持信息</a>
                    </div>
                </li>
                <li class="nav-item dropdown">
                    <a class="nav-link en" href="#">Wet lab</a>
                    <a class="nav-link cn d-none" href="#">湿实验室</a>
                    <div class="dropdown-menu">
                        <a class="dropdown-item en" href="./protocols.html" i18n="protocos">Protocos</a>
                        <a class="dropdown-item cn d-none" href="./protocols.html" i18n="protocos">实验方案</a>
                        <a class="dropdown-item en" href="./notebook.html" i18n="notebooks">Notebooks</a>
                        <a class="dropdown-item cn d-none" href="./notebook.html" i18n="notebooks">实验记录</a>
                        <!-- <a class="dropdown-item en" href="./results.html">Results</a>
                        <a class="dropdown-item d-none cn" href="./results.html">结果</a> -->
                    </div>
                </li>
                <li class="nav-item dropdown">
                    <a class="nav-link en" href="./team.html" i18n="team">Team</a>
                    <a class="nav-link cn d-none" href="./team.html" i18n="team">团队</a>
                </li>

                <li class="nav-item dropdown">
                    <!-- 月亮图标 -->
                    <i class="theme-toggle">
                        <?xml version="1.0" standalone="no"?>
                        <!DOCTYPE svg
                            PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg
                            t="1695295953556" class="moon" viewBox="0 0 1024 1024" version="1.1"
                            xmlns="http://www.w3.org/2000/svg" p-id="11622" xmlns:xlink="http://www.w3.org/1999/xlink"
                            width="200" height="200">
                            <path
                                d="M529.611373 1023.38565c-146.112965 0-270.826063-51.707812-374.344078-155.225827C51.74928 764.641808 0.041469 639.826318 0.041469 493.815745c0-105.053891 29.693595-202.326012 88.978393-292.22593 59.38719-89.797526 137.000103-155.942569 232.83874-198.63991 6.041111-4.607627 12.184613-3.788493 18.225724 2.252618 7.576986 4.607627 9.931996 11.365479 6.860244 20.580733C322.677735 83.736961 310.493122 142.202626 310.493122 201.589815c0 135.464227 48.328885 251.474031 144.986656 348.131801 96.657771 96.657771 212.667574 144.986656 348.131801 144.986656 74.541162 0 139.252721-11.365479 194.032283-34.19883C1003.684974 655.799424 1009.726084 656.618558 1015.767195 662.659669c7.576986 4.607627 9.931996 11.365479 6.860244 20.580733C983.104241 786.758417 918.802249 869.286132 829.721465 930.925939 740.743072 992.565746 640.706375 1023.38565 529.611373 1023.38565zM296.772633 55.47685c-79.148789 44.130825-142.324472 105.770633-189.424657 184.919422C60.1454 319.545061 36.595308 404.018219 36.595308 493.815745c0 135.464227 48.328885 251.474031 144.986656 348.131801 96.657771 96.657771 212.667574 144.986656 348.131801 144.986656 97.374513 0 185.636163-25.085968 264.784952-75.360296C873.545115 861.299579 932.829913 793.618661 972.455503 708.32637 925.252927 723.582734 868.937488 731.15972 803.509188 731.15972c-146.112965 0-270.826063-51.707812-374.344078-155.225827C325.647095 472.415878 273.939283 347.70278 273.939283 201.589815 273.939283 158.994865 281.516269 110.256413 296.772633 55.47685z"
                                p-id="11623"></path>
                        </svg>
                    </i>
                    <!-- 太阳图标 -->
                    <i class="theme-toggle dropdown">
                        <svg t="1695295862627" class="sun" viewBox="0 0 1024 1024" version="1.1"
                            xmlns="http://www.w3.org/2000/svg" p-id="9560" xmlns:xlink="http://www.w3.org/1999/xlink"
                            width="200" height="200">
                            <path
                                d="M512 950.857143c241.371429 0 438.857143-197.485714 438.857143-438.857143s-197.485714-438.857143-438.857143-438.857143-438.857143 197.485714-438.857143 438.857143 197.485714 438.857143 438.857143 438.857143z m0 73.142857c-285.257143 0-512-226.742857-512-512s226.742857-512 512-512 512 226.742857 512 512-226.742857 512-512 512z"
                                fill="#333333" p-id="9561"></path>
                        </svg>
                    </i>
                </li>
                <!-- 语言图标 -->
                <li class="nav-item dropdown">
                    <i class="lang-toggle">
                        <svg t="1691029428947" class="translate-icon" viewBox="0 0 1024 1024" version="1.1"
                            xmlns="http://www.w3.org/2000/svg" p-id="10939"
                            data-spm-anchor-id="a313x.search_index.0.i13.76c63a81JzoK4S"
                            xmlns:xlink="http://www.w3.org/1999/xlink" width="200" height="200">
                            <path
                                d="M358.912 223.914667h84.309333c17.442133 0 31.3344 14.336 31.3344 31.812266v76.1856a31.4368 31.4368 0 0 1-31.368533 31.3344h-84.2752v95.914667a18.944 18.944 0 0 1-18.807467 18.773333 18.944 18.944 0 0 1-18.8416-18.773333v-95.914667H236.987733a31.8464 31.8464 0 0 1-31.368533-31.778133V255.2832c0-16.9984 13.892267-31.3344 31.402667-31.3344h84.241066v-33.1776c0-10.308267 8.533333-18.773333 18.8416-18.773333 10.308267 0 18.807467 8.465067 18.807467 18.773333v33.1776zM243.268267 325.632h77.994666V262.007467H243.268267V325.632z m115.643733 0h78.0288V262.007467h-78.0288V325.632z m304.810667 396.936533l34.952533-88.2688 34.065067 88.2688h-69.0176z"
                                fill="#10656B" p-id="10940" data-spm-anchor-id="a313x.search_index.0.i10.76c63a81JzoK4S"
                                class=""></path>
                            <path
                                d="M408.234667 615.936H133.461333a69.188267 69.188267 0 0 1-69.461333-69.4272V133.4272c0-38.5024 30.9248-69.4272 69.461333-69.4272h413.2864c38.536533 0 69.4272 30.9248 69.4272 69.4272v275.114667h265.386667a78.506667 78.506667 0 0 1 78.4384 78.370133v394.683733a78.506667 78.506667 0 0 1-78.4384 78.404267H486.673067a78.506667 78.506667 0 0 1-78.4384-78.404267v-265.659733z m39.253333-39.185067l129.501867-129.501866V133.461333a30.071467 30.071467 0 0 0-30.242134-30.242133H133.461333a30.037333 30.037333 0 0 0-30.242133 30.242133v413.0816c0 16.861867 13.346133 30.242133 30.242133 30.242134h314.026667z m354.304 221.525334L716.117333 575.146667a18.978133 18.978133 0 0 0-17.476266-12.117334 18.909867 18.909867 0 0 0-17.476267 12.117334l-88.302933 223.095466a18.807467 18.807467 0 0 0 10.752 24.644267 18.807467 18.807467 0 0 0 24.644266-10.752l20.1728-51.541333h98.2016l19.694934 51.541333a18.7392 18.7392 0 0 0 24.644266 10.752 18.773333 18.773333 0 0 0 10.752-24.644267zM716.117333 176.913067a19.114667 19.114667 0 0 1-17.476266-20.616534 19.285333 19.285333 0 0 1 20.1728-17.476266c57.344 4.471467 167.185067 46.148267 170.325333 171.588266a18.602667 18.602667 0 1 1-37.205333 0.887467c-3.584-121.856-130.423467-133.9392-135.816534-134.382933z"
                                fill="#10656B" p-id="10941"></path>
                            <path
                                d="M318.122667 830.976a19.0464 19.0464 0 0 1 6.485333 34.338133 18.602667 18.602667 0 0 1-13.653333 2.833067c-140.458667-27.136-162.474667-160.699733-164.317867-172.1344l-0.170667-0.785067a19.0464 19.0464 0 0 1 16.622934-21.060266 19.012267 19.012267 0 0 1 21.026133 16.5888l0.1024 0.682666c1.706667 10.5472 18.944 116.906667 133.9392 139.537067z"
                                fill="#10656B" p-id="10942"></path>
                        </svg>
                    </i> <!-- 语言图标 -->
                </li>

            </ul>
        </div>
    </nav>
    <!-- 侧边栏 -->
    <!-- <img src="../asserts/img/sidebar.png" alt="sidebar" class="sidenav-top-img"> -->
    <div class="sidenav sidenav-bg toptransparent">
        <svg id="mySVG" class="mySVG-sidebar">
            <circle fill="none" cx="120" cy="115" r="67" stroke="#F29700" stroke-width="5" id="triangle"
                transform="rotate(270 120 115)"></circle>
            <circle fill="#363636" cx="120" cy="115" r="63" stroke="#363636" stroke-width="5" / class="circle-SVG">
                Sorry, your browser does not support inline SVG.
        </svg>

        <a class="navbar-brand justify-content-center" href="../index.html">
            <img src="../asserts/img/sidenav-logo.svg" alt="logo" class="sidenav-logo logo justify-content-center">
        </a>
        <div>
            <h2 class="justify-content-center" style="color:#F29700; display:flex;margin-bottom:30px;margin-top:150px;">
                Notebooks
            </h2>

            <a href="#overview" style="margin-top: -20px;" class="" i18n="overview">Overview</a>
            <a href="#00" class="">Introduction</a>
            <a href="#01" class="">Theoretical basis</a>
            <a href="#02" class="">Conductivity optimization model</a>
            <a href="#03" class="">Model Overview</a>
            <a href="#04" class="">Molecular Dynamics Simulations</a>
            <a href="#05" class="">Optimization Design</a>
            <a href="#06" class="">Semi-rational Directed Evolution </a>
            <a href="#07" class="">Future work</a>
            <a href="#08" class="">Reference</a>
            <a href="#overview" class="">Back to top ⬆ </a>
        </div>
    </div>
    <!-- 主要内容 -->
    <article class="main">
        <div id="overview" class="section-header">
            <img src="../asserts/img/myoverview.png" alt="overview_bg" class="img-fluid " style="width:100vw;"></img>
            <h1 class="leader img-fluid">Model</h1>
        </div>

        <div id="overview" class="section header" style="padding-top: 4rem;">
            <div id="" class="" style="min-height: 10vh; margin-bottom: 0px;">
                <div class="overview-bg">
                    <div class="overview-logo">
                        <img id="overview" src="../asserts/img/overview.png" class="overview-img">
                    </div>
                    <p class="en">This year, we used a special bio-nanomaterial derived from Geobacter metallireducens,
                        as the "variable resistor" in our disease biomarker detection system. By detecting the changes
                        in conductivity of the e-pili at different concentrations of the target substance and combining
                        it with a standard curve, we can achieve quantitative detection of the target substance (To
                        learn more: <a href="./design.html">design</a>).</p>
                    <p class="cn d-none">
                        在我们今年的项目中，我们将一种特殊的生物纳米材料---来源于金属还原地杆菌的Ⅳ型菌毛---作为我们的疾病标志物检测系统中的“可变电阻”，通过检测其在不同待测物浓度下的电导率变化并结合标准曲线，我们就能实现对于待测物的定量检测（To
                        learn more：<a href="./design.html">design</a>）。</p>
                </div>
            </div>
            <!-- Header Section Content -->
        </div>
        <div class="container">
            <h2 id="00" class="module">Introduction</h2>
            <p class="en">In our system, the conductivity of the e-pili is the most important parameter. Higher
                conductivity allows our detection system to become more stable and accurate. Additionally, higher
                conductivity may also imply more possibilities for future applications of this bio-nanomaterial.</p>
            <p class="cn d-none">在我们的系统中，导电菌毛的电导率是最重要的一个指标。更高的电导率可以让我们的检查系统变得更加的稳定和精确。除此之外，更高的电导率还意味着更多应用的可能性。</p>
            <p class="en">In this section，we use optimization methods of mathematical modeling to optimize the structure
                of this pili to improve its conductivity.Conbing with an innovative screening method based on the
                microbial fuel cell system,we may eventually obtain a pili with higher conductivity.</p>
            <h3 class="en">1. Lower current thermal effect</h3>
            <h3 class="cn d-none">1.更低的电流热效应：</h3>
            <p class="en">Compared to inorganic conductive nanomaterials, an important point to note about e-pili is
                their chemical nature as proteins.This means that we need to consider whether they can function stably
                in the working environment of traditional inorganic materials.</p>
            <p class="en">In our system, the e-pili act as variable resistors. When current passes through the e-pili,
                work is done, resulting in heat proportional to the resistance of the e-pili. Since our system requires
                measuring the conductivity changes of the nanowires over a period of time, it implies that the e-pili
                will be constantly exposed to the Joule heating effect. Being protein-based materials, long-term
                exposure to high temperatures may cause denaturation (by using DeepSTABp tool,we estimated the melting
                temperature of these nanowires to be 43 degrees). This could potentially affect the stability and
                accuracy of the system.</p>
            <p class="cn d-none">
                相较于无机的导电纳米材料，一个值得我们注意的点是导电菌毛的化学本质是一种蛋白质，这就意味着我们需要考虑其是否能在传统的无机材料的工作环境下稳定的工作。导电菌毛在我们的系统中作为一个可变电阻，而当电流通过菌毛时，电流会做功并产生和菌毛的阻值成正比的热量。由于我们的系统需要测量菌毛在一段时间内的电导率变化曲线，这就意味着此时导电菌毛会一直处在电流热效应中。而导电菌毛作为一种蛋白质材料，长期处于较高的温度中可能会使其发生变性（我们使用DeepSTABp，简单估计了一下这种菌毛的热变性中点温度为43度），从而影响到系统的稳定性和精确程度。
            </p>
            <h3 class="en">2.Smoother resistance variation curve</h3>
            <h3 class="cn d-none">2.更平滑的电阻变化曲线：</h3>
            <p class="en">We have found that the resistance of e-pili with higher conductivity has a smoother curve when
                pH changes. The range of conductivity changes within the measured range is also greater, which may help
                improve the detection threshold. For example, compared to the wild-type , W51W53 (a mutant of Geobacter
                sulfurreducens with higher conductivity) shows a stronger linear relationship in the change of its
                conductivity as the external pH varies from 2.0 to 10.5 [1]. A stronger linear relationship means that
                we can better establish models and quantify and analyze experimental results.</p>
            <p class="cn d-none">
                我们发现，具有更高电导率的菌毛的电阻随pH的变化曲线更加的平滑，在测量范围内的电导率变化的范围也更大（可能有助于提高检测的阈值）。例如相较于野生型的硫还原地杆菌的菌毛，W51W53（一种硫还原地杆菌菌毛的突变体，具有更高的电导率）在外界的pH值从2.0到10.5的过程中其电导率的变化的线性性明显更强[1]。更强的线性性就意味着我们可以更好的建立模型以及量化和分析实验结果。
            </p>
            <h3 class="en">3.Further unlocking its application potential</h3>
            <h3 class="cn d-none">3.进一步解放其应用潜力：</h3>
            <p class="en">In addition to the lack of mature large-scale production techniques, one of the reasons that
                currently limits the application of these conductive biological nanomaterials is their conductivity
                compared to traditional inorganic nanomaterials such as carbon nanotubes and silicon nanotubes still at
                a disadvantage[1]. However, by improving the conductivity of e-pili, we can further unlock the
                application potential of this green biogenic nanomaterial.</p>
            <p class="cn d-none">
                除此之外，作为一种导电生物纳米材料，目前限制其投入应用的原因除了缺少成熟的大规模生产技术外，这种材料的电导率相较于传统的无机纳米材料例如碳纳米管和硅纳米管还有着一定的差距[1]。而提高菌毛的电导率毫无意味可以进一步的解放这种绿色生物纳米材料的应用潜力。
            </p>
            <p class="en">After clarifying our objectives, we extensively reviewed existing literature but found no
                studies related to optimizing the conductivity of e-pili. </p>
            <p class="cn d-none">在明确了我们要做的事情后，我们查阅了大量的资料但都没有发现和优化导电菌毛电导率相关的研究。</p>
            <p class="en">An idea came to our minds: since no one has done it before, let's give it a try.</p>
            <p class="cn d-none">一个想法出现在了我们的脑海里：既然没有别人做过，那就让我们来试试吧。</p>
            <h2 id="01" class="module">Theoretical basis</h2>
            <p class="en">Within e-pili, there is a high density of aromatic amino acids (Figure 1). Current research
                suggests that the conductivity of e-pili mainly originates from the overlapping Π-Π orbitals of aromatic
                rings formed by the aromatic moieties of aromatic amino acid residues in their unique structure[2].
                Numerous studies based on this theoretical foundation have made significant progress [1,3,4].</p>
            <p class="cn d-none">
                在导电菌毛中分布着高密度的芳香族氨基酸（图一）。目前绝大部分研究都认为，导电菌毛的电导率主要来源于其特殊结构中芳香族氨基酸残基上的芳香环形成的电子云轨道的交叠[2]。有大量基于该理论基础的研究都取得了的进展[1,3,4]。
            </p>
            <p class="en">Further consideration suggested that not only aromatic amino acid abundance, but also the
                position and spacing of aromatic amino acids in the pilin could be important[5]. For example, there is a
                stretch of 53 amino acids without an aromatic amino acid within The G. uraniireducens pilin. This
                aromatic-free gap might prevent close-packing of aromatic amino acids within the assembled pili.
                Heterologous expression in G. sulfurreducens of other type IV pilins with high aromatic amino acid
                abundance and smaller aromatic-free gaps yielded e-pili that appeared to be as conductive as wild-type
                G. sulfurreducens e-pili and functioned well in extracellular electron transfer[5]. </p>
            <p class="cn d-none">
                菌毛的高分子结构拥有延长共轭双键，离域π键电子不受原子束缚，能在聚合链上自由移动，经过掺杂后，可移走电子生成空穴，或添加电子，使电子或空穴在分子链上自由移动，从而形成导电分子。</p>
            <p class="en">This means that if each aromatic amino acid in a sequence within the e-pili can form π-π
                interactions with adjacent aromatic amino acids, an effective conductive pathway is likely to form. At
                this point, the distance between aromatic amino acids and the rate of electron transfer exhibit a
                negative correlation.</p>
            <p class="cn d-none">这就意味着如果导电菌毛的某一串芳香族氨基酸中的每个芳香族氨基酸的芳香环均可以与其邻近的芳香族氨基酸的芳香环形成 π-π
                的相互作用，则这串芳香族氨基酸就可能形成有效的导电通路。而此时芳香族氨基酸之间的距离与电子转移速率大小呈现负相关关系。</p>
            <p class="en">However, when the external pH changes or other molecules are adsorbed onto the surface of
                e-pili, the orientation and spacing of the aromatic rings of the aromatic amino acids on the surface may
                change due to alterations in surface charge distribution. Consequently, this leads to changes in the
                conductivity of e-pili.</p>
            <p class="cn d-none">
                简单的说，导电菌毛中能够形成ΠΠ堆积的芳香环越密集，菌毛的电导率越高。而当外界的pH值发生变化或导电菌毛表面吸附上其他分子的时候，导电菌毛表面的芳香族氨基酸的苯环的朝向和间距会因为表面电荷分布情况的改变而发生变化，进而改变导电菌毛的电导率。
            </p>
            <div class="row">
                <div class="col-md-3"></div>
                <img src="../asserts/img/model/model_1.png " alt="" class="col-md-6">
                <p>Fig1. Schematic illustration of the principle of conductivity/electrical conductivity changes in
                    e-pili.
                </p>
                </img>
            </div>
            <h2 id="02" class="module">Conductivity optimization model</h2>
            <p class="en">
                To establish an optimization model for the conductivity of e-pili, we need to consider three basic
                elements of optimization separately: <strong>Decision variables</strong>, <strong>Objective
                    function</strong>, and <strong>Constraints</strong>.
            </p>
            <p class="cn d-none">
                要建立导电菌毛的电导率最优化模型，我们需要分别考虑最优化类模型的三个基本要素：决策变量，目标函数和约束条件。
            </p>
            <h3>Decision variables</h3>
            <p class="en">
                <strong>The decision variables are factors that decision-makers can control.</strong><br>
                In this model, the variables we can control are the amino acid sequences of the e-pili. By changing the
                types and positions of amino acids in the sequence, we can attempt to alter the overall conductivity of
                the pili. The most critical indicator affecting pili conductivity is the spacing of Π residues (aromatic
                amino acids capable of producing Π-Π interactions). There are two methods to modify this spacing:
            </p>
            <p class="cn d-none">
                <strong>决策变量是决策者可以控制的因素。</strong><br>
                在这个模型中，我们能控制的变量是金属还原地杆菌的导电菌毛的氨基酸序列。我们可以通过改变序列中的氨基酸的种类和位置来尝试改变整体菌毛的电导率。影响菌毛电导率的最核心的指标是其中Π残基（能够产生ΠΠ相互作用的芳香族氨基酸称作Π残基）的间距。而改变其间距的方法有以下两种：
            <h3 class="en">1.Altering the quantity and position of aromatic amino acids within e-pili.</h3>
            <h3 class="cn d-none">1.改变导电菌毛内芳香族氨基酸的数量和位置</h3>
            <div class="row">
                <div class="col-md-2"></div>
                <img src="../asserts/img/model/model_2.png " alt="" class="col-md-8">
                <p>Fig2. Alterations in the quantity and position of aromatic amino acids within the e-pili.
                </p>
                </img>
            </div>
            <p class="en">The quantity and positioning of aromatic amino acids within the e-pili are important factors
                determining their conductivity. For example, in comparison to the wild type G. sulfurreducens's e-pili,
                a mutant variant Y27A exhibits less than one-fifth of the conductivity. This is because in Y27A,
                tyrosine, an aromatic amino acid, is replaced by alanine, a non-aromatic amino acid, leading to an
                increased spacing between the Π residues in that specific region. This alteration impacts the transfer
                of electrons, resulting in reduced conductivity.</p>
            <p class="cn d-none">
                芳香族氨基酸的在导电菌毛内部的数量和位置是决定其电导率的一个重要原因。例如相较于野生型的硫还原地杆菌的菌毛，其突变体Y27A的电导率只有其不到5分之一。这是因为Y27A用一个丙氨酸代替了芳香族氨基酸酪氨酸，使得局部的Π残基之间的间距变大，影响到了电子的传递。
            </p>
            <h3 class="en">2.
                Modifying the type of amino acids to alter the overall hydrophilicity and hydrophobicity of the e-pili:
            </h3>
            <h3 class="cn d-none">2.改变氨基酸的种类以改变整个菌毛亲疏水性：</h3>
            <div class="row">
                <div class="col-md-2"></div>
                <img src="../asserts/img/model/model_3.png " alt="" class="col-md-8">
                <p>
                    Fig3. Impact of amino acid type alteration on the electrical conductivity of e-pili.
                </p>
                </img>
            </div>
            <p class="en">In addition, the hydrophobicity of amino acids in the e-pili can also influence the overall
                structure and spacing of Π residues within the e-pili. For instance, in the mutant W51W57, replacing the
                phenylalanine and tyrosine at the c-terminal of the pili with tryptophan enhances the hydrophobicity of
                the pili and reduces the diameter of the pili. This results in a more compact structure, leading to a
                reduction in the spacing between internal Π residues. As a result, the conductivity of W51W57 has been
                shown to be approximately 2000 times higher than that of the wild type in some studies[6].</p>
            <p class="cn d-none">
                除此之外，导电菌毛中氨基酸的亲疏水性可以影响到菌毛的整体结构从而导致内部的Π残基间距发生变化。例如在突变体W51W57中，通过用色氨酸替代导电菌毛羧基端的苯丙氨酸和酪氨酸，使得组装后菌毛的疏水性增强，菌毛的直径也变得更小。此时导电菌毛的结构变得更加紧凑，内部Π残基的间距也明显减小。这使得W51W57的电导率在一些研究中比野生型的电导率提高了大约2000倍[6]。
            </p>
            <p class="en">Due to the complexity of the structural model for type IV pili assembly and the lack of
                suitable methods to predict the impact of hydrophobicity changes in individual pili subunits on the
                overall assembled pili structure, our focus in this model is primarily on <strong>method one:</strong>
                understanding the impact of introducing new aromatic amino acids or altering the positions of existing
                aromatic amino acids on the pili conductivity.</p>
            <p class="cn d-none">
                由于Ⅳ型菌毛的结构模型构建相对复杂，且目前我们也没有找到合适的方法来预测菌毛单体的疏水性变化对于整个组装后菌毛结构的影响。在本模型中我们主要关注<strong>方法一:</strong>即增加新的芳香族氨基酸或者改变原有芳香族氨基酸的位置对于菌毛电导率的影响。
            </p>
            <p class="en">Furthermore, it's worth noting that the assembly of e-pili requires a complex and conserved
                type IV pilus assembly system. This assembly system has certain requirements for the conservation of
                amino acid sequences within the pili[7]. Modifying too many amino acids may not only result in the
                collapse of the original pili structure but also prevent proper assembly within cells.</p>
            <p class="cn d-none">
                除此之外，值得注意的是，金属还原地杆菌的导电菌毛需要一套复杂且保守的Ⅳ型菌毛组装系统来进行组装。而这套组装系统对于菌毛的氨基酸序列的保守性是有一定的要求的。如果修改的氨基酸过多不仅可能会导致原有菌毛结构的崩溃还可能导致其无法在工程菌体内正确组装。
            </p>
            <p class="en">This implies that our algorithm needs to find the optimal modification strategy under the
                constraint of minimizing the number of modifications required.</p>
            <p class="cn d-none">这就意味着我们的算法需要能够在使用最少修改次数的条件下找到此时的最优修改方案。</p>
            <h3>Objective function</h3>
            <p class="en"><strong>The objective function is a function used to describe the goals pursued by
                    decision-makers.</strong></p>
            <p class="cn d-none"><strong>目标函数是用于描述决策者所追求的目标的函数。</strong></p>
            <p class="en">In our model, our ultimate goal is to obtain e-pili with higher conductivity characteristics.
                The conductivity characteristics of the e-pili are closely related to the distance between the centroid
                of the aromatic ring on the aromatic amino acid residues.</p>
            <p class="cn d-none">在我们的模型中，我们最终的目标是得到具有更高的电导率特性的导电菌毛。而导电菌毛的电导率特性和其中芳香族氨基酸残基上的芳香环的质心之间的间距息息相关。</p>
            <p class="en">Based on this theory, we have designed the following two objective functions:</p>
            <p class="cn d-none">基于这个理论，我们设计了如下两个目标函数：</p>
            <h3 class="en">1. Average distance between the centroid of aromatic rings of Π residues on the shortest
                electron transfer chain in e-pili:</h3>
            <h3 class="cn d-none">1.导电菌毛中最短电子传递链上的Π残基芳香环质心平均间距：</h3>
            <div class="row">
                <div class="col-md-2"></div>
                <img src="../asserts/img/model/model_4.png " alt="" class="col-md-8">
                <p>
                    Fig4. Schematic representation of the shortest electron transfer chain.
                </p>
                </img>
            </div>
            <p class="en">This objective function is used to describe the maximum conductivity of conductive pili under
                ideal conditions.</p>
            <p class="cn d-none">该目标函数用于描述在理想状态下导电菌毛的最高电导率。</p>
            <p class="en">We can assume that in an ideal state, electrons will move along the shortest propagation path
                composed of Π residues in e-pili. At this point, the conductive pili has the highest conductivity within
                this structure.</p>
            <p class="cn d-none">
                我们可以认为在理想状态下电子会沿着导电菌毛中由Π残基构成的最短传播路径移动。此时导电菌毛具有该结构下的最高的电导率。该函数描述了给定结构条件下菌毛电导率的上限。以该函数为目标的优化可以让我们得到具有更高电导率的菌毛。
            </p>
            <p class="en">within this structure.
                This function describes the upper limit of the conductivity of pili under given structural conditions.
                Optimization based on this objective function may help us obtain the pili with higher conductivity.</p>
            <h3 class="en">2.Average length between neighboring nodes of the Minimum Spanning Tree for all Π residues.
            </h3>
            <h3 class="cn d-none">2.所有Π残基节点最小生成树中的相邻节点平均间距：</h3>
            <div class="row">
                <div class="col-md-2"></div>
                <img src="../asserts/img/model/model_5.png " alt="" class="col-md-8">
                <p>
                    Fig5. Schematic illustration of the minimum aromatic amino acid node spanning tree.
                </p>
                </img>
            </div>
            <p class="en">This objective function is used to describe the overall conductivity of e-pili under the
                influence of external environmental factors.</p>
            <p class="cn d-none">该目标函数用于描述在受外界环境影响的条件下导电菌毛的综合电导率。</p>
            <p class="en">Under the influence of environmental factors such as pH changes or surface adsorption, other
                aromatic amino acids in e-pili, beyond the original shortest electron propagation path, may also
                participate in the electron transfer process as the structure of the e-pili changes. </p>
            <p class="cn d-none">
                在环境因素（例如pH变化或者表面吸附）的影响下导电菌毛中原有的最短电子传播路径之外的其他芳香族氨基酸也可能随着菌毛结构的变化参与到电子传递的过程中。该目标函数描述了所有菌毛内部的Π残基之间的间隔情况，可以反应在不同环境因素影响下菌毛的综合导电能力。以该函数为目标的优化可能使得我们的菌毛在不同环境条件下的电导率表现更好，电导率曲线也可能变得更加平滑。
            </p>
            <p class="en">This objective function describes the spacing between all Π residues inside the pili,
                reflecting the overall conductivity of the pili under different environmental influences. Optimization
                based on this objective function may improve the conductivity performance of our pili under different
                environmental conditions and make the conductivity curve smoother.</p>
            <h3>Constraints</h3>
            <p class="en"><strong>Constraints are the limitations that decision variables must satisfy.</strong></p>
            <p class="cn d-none"><strong>约束条件是决策变量需要满足的限定条件。</strong></p>
            <p class="en">In this model, the constraints are the possible coordinates of the centroid of aromatic rings
                in aromatic amino acids. All decision variables (the added aromatic amino acids) can only occur at
                specific positions determined by the structure of e-pili themselves. </p>
            <p class="en">Since we only focus on the changes in conductivity caused by the quantity and position of
                aromatic amino acids , we can initially assume that adding or modifying a small number of aromatic amino
                acids has minimal impact on the overall structure of the e-pili. This means that we do not need to
                rebuild the molecular dynamics model for each iteration. Instead, we can use the spatial coordinate
                information of the original structure to roughly determine the range of the centroid coordinates for the
                new aromatic rings.</p>
            <p class="cn d-none">
                在本模型中，约束条件是芳香族氨基酸的芳香环质心的可能坐标，所有的决策变量（新增的芳香族氨基酸）都只能出现在由导电菌毛结构本身决定的特定位置上。由于我们在本模型中我们只讨论芳香族氨基酸的数量和位置对于其电导率的影响（思路一），在此我们可以先假设添加或修改少量芳香族氨基酸对于菌毛整体结构的影响不大，这就意味着我们不需要每进行一次迭代就需要重新完成一遍分子动力学模型的构建，而是可以用原有结构的分子空间坐标信息来大致给出新的芳香环的质心坐标的范围。
            </p>
            <p class="en"><strong>In our model, we approximate the centroid coordinates of the newly added aromatic ring
                    by using the average coordinates of all carbon atoms in the side chain at the modified
                    position.</strong></p>
            <p class="cn d-none"><strong>在我们的模型中，我们用所改动位置侧链上所有碳原子的平均坐标来近似预测新增的芳香环的质心坐标。</strong></p>
            <p class="en">Since the purpose of constructing this model is to discover potential mutation sites, we can
                combine the semi-rational directed evolution method to screen and obtain the actual optimal results.
                Therefore, the accuracy requirement for this model is not as strict as that of rational protein design.
            </p>
            <p class="cn d-none">由于本模型的构建旨在发掘可能的突变位点，我们可以结合蛋白质半理性定向进化的方法来筛选得到实际的最优结果，故本模型中对于模型精度的要求并不如蛋白质理性设计那样严格。</p>
            <p class="en">Since the purpose of constructing this model is to discover potential mutation sites, we can
                combine the semi-rational directed evolution method to screen and obtain the actual optimal results.
                Therefore, the accuracy requirement for this model is not as strict as that of rational protein design.
            </p>
            <p class="cn d-none">由于本模型的构建旨在发掘可能的突变位点，我们可以结合蛋白质半理性定向进化的方法来筛选得到实际的最优结果，故本模型中对于模型精度的要求并不如蛋白质理性设计那样严格。</p>

            <h2 id="03" class="module">Model Overview</h2>
            <div class="row">
                <img src="../asserts/img/model/model_6.jpg " alt="" class="col-md-12">
                </img>
            </div>
            <h2 id="04" class="module">Molecular Dynamics Simulations</h2>
            <p class="en">Common methods to obtain pili protein structure include nuclear magnetic resonance imaging,
                X-ray technology and homology prediction. These methods have some problems such as static snapshot,
                signal interference and atomic position conflict. And when we use spiral symmetric assembly for pili
                monomers, it is easy to overlook the flexibility of subunits. Therefore, we first use amber23 for
                molecular dynamics optimization of the subunits, and then use Rosetta for helical symmetric assembly of
                the optimized subunits with fuzzy constraints. To eliminate the effects of rigid docking, molecular
                dynamics simulation (amber23) was used again to optimize the pili assembly structure to obtain the best
                pili polymer structure. Finally, pymol was used to further verify the obtained pili structure, including
                distance and electron transport chain. The use of molecular dynamics to optimize the assembly process
                has obvious advantages: the protein is simulated in a real biological environment and the dynamic
                effects of the organism are taken into account. Molecular dynamics simulation (MD) can be used to
                optimize the structure of pili proteins and annotate the structural details.</p>
            <p class="cn d-none">通常获得菌毛蛋白结构方法包括：核磁共振成像，X
                射线技术和同源预测等。这些方法存在静态快照、信号干扰和原子位置冲突等问题。并且我们对于菌毛单体使用螺旋对称组装时，容易忽视亚基的柔性。所以，我们首先使用amber23对于亚基进行分子动力学优化，再使用Rosetta对优化后的亚基进行增加了模糊条件约束的螺旋对称组装。为了消除刚性对接的影响，我们再次运用分子动力学模拟(amber23）优化菌毛组装结构以获得最佳的菌毛聚合物结构。最终，使用pymol对于得到的菌毛结构进行进一步的验证，包括距离、电子传递链等。使用分子动力学去优化组装的过程是有很明显的优势：蛋白模拟在一个真实的生物学环境以及考虑了生物体的动力学效应。运用分子动力学模拟（MD)，可以优化菌毛蛋白的结构和注释结构细节。
            </p>
            <div class="row">
                <img src="../asserts/img/model/model_7.png " alt="" class="col-md-12">
                </img>
            </div>
            <h2 id="05" class="module">Optimization Design</h2>
            <p class="en">The determination of the e-pili structure implies the establishment of constraints in our
                optimization model. Next, we will design corresponding optimization functions for the optimization
                objectives we previously proposed and use algorithms to optimize each of them separately.</p>
            <p class="cn d-none">菌毛结构的确定意味着我们优化算法中的约束条件也随之确定，接下来我们要针对我们之前提出的优化的目标分别设计对应的优化函数，并使用算法分别进行优化。</p>
            <h3>Graph structure construction</h3>
            <p class="en">We create a new data class for pi-residues to store all their information, with the average of
                the x, y, and z coordinates of all the carbon atoms in each pi-residue as its center-of-mass coordinates
                in the three-dimensional space. We then construct a graph with each pi-residue as a vertex and obtain
                the corresponding adjacency and distance matrices based on the center-of-mass coordinates. Among them,
                we set a value k to constrain that only pi-residues with a distance less than k are capable of
                transferring electrons, otherwise, we set their adjacency value to 0 and their distance to infinity.</p>
            <p class="cn d-none">
                我们首先创建新的数据结构并储存了全部的pi残基信息，将每个pi残基中碳原子x，y，z轴坐标的平均值视为该pi残基在三维空间中的质心坐标。我们将得到的pi残基结构视为以每一个pi残基为顶点的“图”，并基于所求的质心坐标得到相应的邻接矩阵和距离矩阵。其中，我们认为只有两个距离小于k的pi残基才是可以完成电子传递的，否则，我们设定其邻接值为0，距离为无穷。
            </p>
            <h3>For Objective function One</h3>
            <p class="en">
                Our first optimization target is <strong>the average distance between the centroids of Π-residue
                    aromatic rings on the shortest electron transfer chain in e-pili</strong>. This optimization target
                is used to describe the highest electrical conductivity of conductive pili under ideal conditions. To
                obtain the shortest electron transfer chain within the e-pili, we can view the network composed of
                Π-residue（aromatic amino acids capable of forming π-π interactions）nodes as a map and use an algorithm
                for identifying the shortest paths between nodes in the map to explore possible electron transfer
                pathways within a series of models.
            </p>
            <p class="cn d-none">
                我们的第一个优化目标是导电菌毛中最短电子传递链上的Π-残基芳香环质心之间的平均间距，这一优化目标用于描述导电菌毛在理想条件下的最高电导率。为了获得菌毛内部的最短的电子传递链，我们可以将由Π-残基节点构成的网络看作是一张地图，并使用用于识别地图中各节点之间最短路径的算法来探索一系列的模型中存在的可能的电子传递路径。
            </p>
            <p class="en">
                The <strong>Dijkstra algorithm</strong> is a commonly used graph search and shortest path algorithm in
                computer science and mathematics. This algorithm aims to find the shortest paths between a starting node
                (or vertex) and all other nodes in a weighted graph, where each edge has an associated numerical weight
                or cost.
            </p>
            <p class="cn d-none">
                Dijkstra算法是计算机科学和数学中常用的图搜索和最短路径算法。该算法旨在找到加权图中起始节点（或顶点）与所有其他节点之间的最短路径，其中每条边都有与其相关的数字权重或代价。</p>
            <p class="en">
                Based on the Dijkstra algorithm, we have designed an algorithm that can search for possible shortest
                electron transfer chains within the e-pili. This algorithm consists of three steps: <strong>(1)
                    Identifying Π-residues；(2) extracting the coordinates of Π-residues；(3) calculating the distances
                    between Π-residues and searching and comparing to determine the shortest path.</strong>
            </p>
            <p class="cn d-none">我们基于Dijkstra 算法设计了一套可以搜索菌毛内部可能的最短电子传递链的算法，该算法由三个步骤组成：（1）识别 π-残基（能够形成 π-π
                相互作用的芳香族氨基酸）（2）提取π-残基的坐标（3）计算π-残基之间的距离并搜索并比较得出最短路径。</p>

            <div class="accordion-container" style="margin-top: 2rem;">
                <button class="accordion">Dijkstra's Shortest Path Algorithm</button>
                <div class="panel">
                    <h3 style="margin-top: 2rem;">Inputs:</h3>
                    <p>
                        Source node, Target node, Distance matrix, Adjacency matrix
                    </p>
                    <h3>Initialization:</h3>
                    <p>
                        Distance array: An array of distances from the source node, initialized as follows:
                    </p>

                    <p>
                        Set the distance from the source node to itself (dist(source)) to 0.
                    </p>

                    <p>
                        Set the distance from all other nodes (v) to infinity (∞) for nodes v not equal to the source
                        node.
                    </p>

                    <p>
                        Q: A queue of all nodes in the graph.
                    </p>

                    <p>
                        S: An empty set to indicate which nodes the algorithm has visited.
                    </p>
                    <p></p>

                    <h3>Proceeding:</h3>
                    <p>
                        While queue Q is not empty, follow these steps:
                    </p>
                    <p></p>

                    <h3>Step1:</h3>
                    <p>
                        Pop the node v from the queue Q that is not already in set S and has the smallest dist(v) value.
                    </p>
                    <p></p>

                    <h3>Step2:</h3>
                    <p>
                        Add node v to set S to indicate that it has been visited.
                    </p>


                    <h3>Step3:</h3>
                    <p>
                        Update the dist values of adjacent nodes of the current node v as follows:
                    </p>

                    <p>
                        For each new adjacent node u, if dist(v) + Distance_matrix[u, v] < dist(u), then update dist(u)
                            to the new minimal value. </p>


                            <h3></h3>
                            <p>
                                The algorithm stops after the target node is visited.
                            </p>


                </div>
            </div>
            <h3 style="padding-top:4rem;">For Objective function Two</h3>
            <p class="en">
                Our second optimization objective is <strong>the average distance between adjacent nodes in the minimum
                    spanning tree formed by all Π-residue nodes.</strong> This optimization objective is used to
                describe the comprehensive conductivity of e-pili under the influence of the external environment.
                Similar to the approach for the first optimization objective, we only need to find the minimum spanning
                tree within the map composed of Π-residue nodes, and then we can obtain the required value by summing
                the weights of the edges in this tree and dividing it by the number of nodes.
            </p>
            <p class="cn d-none">
                我们的第二个优化目标是由所有的Π残基节点构成的最小生成树中的相邻节点的平均间距，该优化目标用于描述受外界环境影响的条件下的导电菌毛的综合电导率。和第一个优化目标的处理方法类似，我们只用求解出由Π残基构成的地图中由各个结点构成的地图中的最小生成树，并将该生成树的中的权值之和除以节点数我们就能得到我们需要的值。
            </p>

            <h3>Prim's Algorithm</h3>
            <p class="en">
                We use Prim's algorithm here to solve this problem. Prim's algorithm is a well-known algorithm used in
                computer science and graph theory to find the minimum spanning tree of a connected, undirected graph
                with weighted edges. The algorithm starts with an initial node and then grows the minimum spanning tree
                one edge at a time until all nodes are included while minimizing the total edge weight.
            </p>
            <p class="cn d-none">
                我们在此使用Prim算法来解决问题，Prim算法是计算机科学和图论中使用的一种著名算法，用于寻找具有加权边的连通无向图的最小生成树。该算法从一个初始节点开始，然后一次一条边增长最小生成树，直到所有节点都被包括在内，同时最小化总边权重。
            </p>

            <div class="accordion-container" style="margin-top: 2rem;">
                <button class="accordion">Prim's Minimum Spanning Tree Algorithm</button>
                <div class="panel">
                    <h3 style="margin-top: 2rem;">Inputs:</h3>
                    <p>
                        Distance matrix
                    </p>

                    <h3>Initialization:</h3>
                    <p>
                        An arbitrary starting node.
                    </p>
                    <p>
                        MST: Initialize an empty set to represent the Minimum Spanning Tree (MST).
                    </p>
                    <p>
                        Key array: Create an array to keep track of the minimum edge weight from each node to the MST.
                        Initialize all values with infinity except for the starting node, which is set to 0.
                    </p>
                    <p>
                        Visited array: Create an array to keep track of visited nodes. Initialize all values to false.
                    </p>

                    <h3>Proceeding:</h3>
                    <p>
                        While the MST set does not include all nodes, follow these steps:
                    </p>

                    <h3>Step1:</h3>
                    <p>
                        Select a node (u) that is not yet in the MST but has the minimum key value.
                    </p>

                    <h3>Step2:</h3>
                    <p>
                        Add node (u) to the MST set.
                    </p>

                    <h3>Step3:</h3>
                    <p>
                        Update key values: For each node v adjacent to the newly added node u, if Distance_matrix[u,v] <
                            Key(v), update the Key(v) with the new weight. </p>

                            <h3>Note:</h3>
                            <p>
                                As a tree structure satisfies:
                            </p>
                            <p>
                                Number of edges = Number of nodes - 1
                            </p>
                            <p>
                                The final objective function is obtained from:
                            </p>
                            <p>
                                (Total length of the MST) / (Number of nodes - 1).
                            </p>
                </div>
            </div>

            <h3 style="padding-top:4rem;">Particle Swarm Optimization</h3>
            <p class="en">Particle Swarm Optimization (PSO) is an optimization algorithm based on swarm intelligence,
                which simulates the behavior of biological swarms, such as flocks of birds or schools of fish, in their
                search for food or resources. </p>
            <p class="cn d-none">粒子群优化算法（Particle Swarm
                Optimization，简称PSO）是一种基于群体智能的优化算法，它模拟了鸟群或鱼群等生物群体在搜索食物或资源时的行为，通过个体之间的协作和信息共享来寻找最优解。</p>
            <p class="en">Particle Swarm Optimization (PSO) can be described as follows: Suppose the search space is
                L-dimensional and there are N particles in the population. The ith particle in the population can be
                represented as an L-dimensional vector \(X_i=(x_{i1},x_{i2},\cdots , x_{il})\), and the best position it
                experiences is denoted as \(P_{best}=(p_{1},p_{2},\cdots , p_{l})\).Each position of the particle
                represents a potential solution to the requirement, which is substituted into the objective function to
                obtain its fitness value. The optimal position searched so far by the whole population is denoted as
                \(G_{best}=(g_{1},g_{2},\cdots , g_{l})\).</p>
            <p class="cn d-none">粒子群优化（PSO）可以描述如下：假设搜索空间是 \(L\) 维的，群体中有 \(N\) 个粒子。群体中的第 \(i\) 个粒子可以表示为一个 \(L\) 维向量
                \(X_i=(x_{i1},x_{i2},\cdots , x_{il})\)，它经历的最佳位置表示为 \(P_{best}=(p_{1},p_{2},\cdots ,
                p_{l})\)。粒子的每个位置代表一个潜在的解决方案，代入目标函数得到其适应值。整个群体迄今为止搜索到的最佳位置表示为 \(G_{best}=(g_{1},g_{2},\cdots , g_{l})\)。
            </p>
            <p class="en">Next, we do the following iterations for each particle in the population :

                $$V_i^{(t+1)} = \omega·V_i^{(t)} + c_1r_1(P_{best}^{(t)} - X_i^{(t)}) + c_2r_2(G_{best}^{(t)} -
                X_i^{(t)})$$

                $$X_i^{(t+1)} = X_i{(t)}+V_i{(t+1)}$$</p>
            <p class="cn d-none">接下来，我们对群体中的每个粒子进行以下迭代：\[V_i^{(t+1)} = \omega V_i^{(t)} + c_1r_1(P_{best}^{(t)} -
                X_i^{(t)}) + c_2r_2(G_{best}^{(t)} - X_i^{(t)})\]

                \[X_i^{(t+1)} = X_i^{(t)}+V_i^{(t+1)}\]
            </p>

            <p class="en">The above formula consists of three parts. The first part is the current speed of the
                particle, with the positive value \(\omega\) being the Inertia Weight, which indicates the current state
                of the particle. The second part is the Cognition Modal, which indicates the cognition of this
                generation of particles about their own state (\(c_1\) is Self-recognition Factor). The third part is
                the Social Modal, which indicates the sharing of information between particles through generations
                (\(c_2\) is Social-recognition Factor).</p>
            <p class="cn d-none">上述公式由三部分组成。第一部分是粒子的当前速度，正值 \(\omega\) 是惯性权重，表示粒子的当前状态。第二部分是认知模式，表示这一代粒子对自己状态的认知（\(c_1\)
                是自我识别因子）。第三部分是社交模式，表示粒子之间通过代代相传的信息共享（\(c_2\) 是社交识别因子）。</p>
            <h3>Discrete Mapping</h3>
            <p class="en">
                In the optimization process, we defined the independent variable as the sequence number of regular
                residues in a monomer that needs to be adjusted to pi-residues. Suppose there are \(N\) non-pi residues
                in each monomer and the number that needs to be changed to pi-residues in each monomer is \(d\), then
                our independent variable X can be denoted as

                \[X=(x_1,x_2,\cdots x_d), x_i \in \{1,2,\cdots N\}\]

                Before performing the optimization, we sorted all the residue sequences of each monomer according to
                their L1 norm from the origin to ensure that the spatial distances of residues with similar sequence
                numbers are also relatively closer spatially, ensuring sequence numbers and the geometrical positions
                are consistent with each other. However, APSO still deals with only the continuous problem, so we need
                to discretize the continuous particles \(Y=(y_1,y_2,\cdots y_d), y_i \in [0,1]\) to correspond to the
                sequence number of residues in each monomer. The method we use is to stick to the continuous
                optimization in the interval [0, 1] and divide [0, 1] into \(N\) equally spaced sub-intervals of length
                \(\frac{1}{N}\). Therefore, we can map \(Y\) to \(X\) one on one by letting \(x_i=[y_i \times N]\),
                which means keeping the integer part of \(Ny_i\).
            </p>

            <p class="cn d-none">
                在优化过程中，我们将独立变量定义为需要调整为π残基的单体中的常规残基的序列号。假设每个单体中有\(N\)个非π残基，每个单体中需要改变为π残基的数量是\(d\)，那么我们的独立变量X可以表示为

                \[X=(x_1,x_2,\cdots x_d), x_i \in \{1,2,\cdots N\}\]

                在进行优化之前，我们根据它们从原点的L1范数对每个单体的所有残基序列进行了排序，以确保具有相似序列号的残基的空间距离也在空间上相对较近，确保序列号和几何位置彼此一致。然而，APSO仍然只处理连续问题，所以我们需要将连续粒子\(Y=(y_1,y_2,\cdots
                y_d), y_i \in [0,1]\)离散化，以对应于每个单体中的残基的序列号。我们使用的方法是坚持在区间[0, 1]中的连续优化，并将[0,
                1]划分为\(N\)个等间距的子区间，长度为\(\frac{1}{N}\)。因此，我们可以通过让\(x_i=[y_i \times
                N]\)将\(Y\)一对一地映射到\(X\)，这意味着保持\(Ny_i\)的整数部分。
            </p>

            <div class="accordion-container" style=margin-top:2rem;">
                <button class="accordion">Adaptive Particle Swarm Optimization</button>
                <div class="panel">
                    <p class="en"><strong>APSO</strong> makes three main improvements on the basis of PSO:</p>
                    <h3>1.Evolutionary State Estimation </h3>
                    <p>ESE calculates the distributional information of the population of each generation in the space
                        of independent variables. We first calculate the average Euclidean distance from particle \(i\)
                        to all the other particles.</p>
                    <div class="row">
                        <div class="col-md-2"></div>
                        <img src="../asserts/img/model/gongshi_1.png " alt="" class="col-md-8">
                        </img>
                    </div>
                    <p>Accordingly, we define the Evolutionary Factor \(f\) that measures the optimizing state of the
                        population:</p>

                    <p>\[f=\frac{d_g-d_{min}}{d_{max}-d_{min}} \in [0,1]\]</p>
                    <p>The Evolutionary factor we calculated above can help to decide which evolutionary state the
                        population is in. The different evolutionary states of the population affect our different
                        adaptive control strategies for the parameters (here we are referring to \(\omega, c_1, c_2\)).
                        We apply the obtained \(f\) to define four different evolutionary states \(S_1,S_2,S_3,S_4\) and
                        their fuzzy membership function, representing the states of exploration, exploitation,
                        convergence, and jumping out respectively.</p>
                    <div class="row">
                        <div class="col-md-2"></div>
                        <img src="../asserts/img/model/model_b3_1.png " alt="" class="col-md-8">
                        </img>
                    </div>
                    <h3>2.Parameters self-adaptive control strategy</h3>
                    <p>We use a sigmoid mapping to allow \( \omega \) to change with the Evolutionary factor \( f \)</p>
                    <p>\[ \omega(f)=\frac{1}{1+1.5e^{-2.6f}} \in [0.4,0.9] \]</p>

                    <p>We know \( \omega(f) \) is a monotonically increasing function. Therefore, a larger \( f \) in \(
                        S_1 \) and \( S_4 \) will result in a larger value of \( \omega \), which facilitates a global
                        search. On the contrary, \( S_2 \) and \( S_3 \) are more suitable for a local search.</p>

                    <p>\( c_1 \) represents self-recognition factor, which pulls the particles to their own historical
                        optimal position; \( c_2 \) represents social-recognition factor, which pushes the population to
                        converge to the current global optimal region and helps fast convergence. These coefficients are
                        initialized to 2.0 and are adaptively controlled according to the evolutionary state with the
                        following strategies:</p>
                    <ol>
                        <p>(1)\( S_1 \) - Increase \( c_1 \) and decrease \( c_2 \) in the exploration state, which
                            helps
                            particles to explore individually and achieve their own historical optimal positions, rather
                            than crowding around the current best particles that may be associated with the local
                            optimum.</p>
                        <p>(2)\( S_2 \) - Slightly increase \( c_1 \) and slightly decrease \( c_2 \) in the
                            exploitation
                            state.</p>
                        <p>(3)\( S_3 \) - Slightly increase \( c_1 \) and slightly increase \( c_2 \) in the convergence
                            state.</p>
                        <p>(4)\( S_4 \) - Decrease \( c_1 \) and increase \( c_2 \) in the jumping-out state.</p>
                    </ol>

                    <h3>3.Elitist Learning Stratrgy</h3>
                    <p>The jumping out effect done by the ELS mechanism on \( G_{best} \) is necessary for the global
                        optimal nature of this algorithm, which randomly selects a dimension of \( G_{best} \), which is
                        represented by the \( g_d \) of the dth dimension:</p>
                    <p>\[ g_d = g_d + (X_d^{max} - X_d^{min})\cdot Gaussian(\mu,\sigma^2) \]</p>

                    <p>APSO's adaptive tuning of parameters can improve the performance of the PSO algorithm in terms of
                        both accelerated convergence and global search. In addition, its ELS strategy will maintain the
                        diversity of the population in order to allow the algorithm to jump out of potential local
                        optima.</p>

                </div>
            </div>
            <h3 style="padding-top:4rem;">Optimization Results</h3>
            <p class="en">We can observe that adding new aromatic amino acids can significantly optimize both our
                objective functions one and two.</p>
            <p class="cn d-none">我们可以观察到，添加新的芳香族氨基酸可以显著的优化我们的目标函数一和目标函数二。</p>
            <p class="en"><strong>In objective function one</strong> , we found that replacing serine with aromatic
                amino acids at position 2 of the original amino acid sequence of the e-pili can reduce the average
                distance in objective one from 4.72 Å to 4.63Å. Furthermore, we found that adding more aromatic amino
                acids on top of this does not further reduce this distance. In objective one, we can consider that the
                electrical conductivity of this mutant has reached its optimal point.</p>
            <p class="cn d-none">
                <strong>在目标函数一中</strong>，我们发现在导电菌毛原有氨基酸序列的2号位将苏氨酸替换成芳香族氨基酸可以将优化目标一中的平均间距从优化前的4.72埃降低到了4.63埃。且我们发现在此基础上加入更多的芳香族氨基酸不会再降低该间距，在优化目标一中，我们可以认为该突变体此时的电导率已经无法再被优化。
            </p>
            <p class="en">To verify the accuracy of the results, we attempted to iterate through all possible spacing
                scenarios for adding 1, 2, and 3 pi-residues (Fig. 7) and found that only the modification at position 2
                changes the original shortest electron transfer pathway, and adding more aromatic amino acids does not
                alter this result.</p>
            <p class="cn d-none">
                为了验证结果的准确性，我们尝试遍历了添加1，2，3个Π残基所有可能的间距情况(Fig.7)，发现只有二号位的修改会改变原有的最短电子传递路径，且添加更多的芳香族氨基酸也不会改变该结果。</p>
            <p class="en"><strong>In objective function two</strong> , because the objective function we chose is used
                to describe the distribution density of aromatic rings within the e-pili, this means that adding 1 or 2
                pi-residues cannot yield an optimal result. Therefore, we started considering the addition of three
                Π-residues, as shown in Fig. 6. Adding 5 aromatic amino acids can reduce the original spacing from 7.6 Å
                to less than 5.5 Å.</p>
            <p class="cn d-none"><strong>在目标函数二中</strong>
                ，由于我们所选用的目标函数用于描述导电菌毛内部芳香环的分布密度，这就意味着添加1，2个Π残基不可能出现最优结果。于是我们从添加三个Π残基开始考虑，结果如Fig6所示。添加5个芳香族氨基酸便可以将原有的间距从7.6埃优化到不到5.5埃。
            </p>
            <div class="row">
                <img src="../asserts/img/model/model_8_1.png " alt="" class="col-md-5">
                <img src="../asserts/img/model/model_8_2.jpg " alt="" class="col-md-7">
                <p>Fig.6 Optimization Results.</p>
                <div class="col-md-1"></div>
                <img src="../asserts/img/model/model_9.png" alt="" class="col-md-10">
                <p>Fig.7 Traversal results after adding 1, 2, and 3 Π-residues (where the x-axis represents the index of
                    the list composed of non-aromatic amino acids, for example, list[0] corresponds to the residue at
                    position 2).</p>
            </div>
            <h3>Model Evaluation</h3>
            <p class="en">Based on the optimization results from the model described above, we reconstructed the
                structural models for single point mutants T2F and triple point mutants A31F, Q53F, and N59F to validate
                our conclusions:</p>
            <p class="cn d-none">基于上述模型优化结果，我们重新构建了点突变体T2F和A31F，Q53F，N59F三位点突变体的结构模型来验证我们的结论：</p>
            <p class="en"><strong>In optimization objective one</strong>, we replaced the threonine at position 2 of the
                wild-type metal-reducing rod bacterium pilin amino acid sequence with phenylalanine. We only selected
                the shortest electron transfer path in the pilus structure obtained by the Dijkstra algorithm (where the
                original aromatic rings are marked in red, and the newly added aromatic rings in mutant T2F are marked
                in yellow). The results are as follows: We observed several gaps in the electron transfer chain obtained
                by the algorithm, which is because the pilin monomers providing the aromatic rings for these gaps are
                not in our model. We counted the distances between the aromatic amino acids on the electron transfer
                chains inside the original wild-type pilus and the mutant pilus. We found that the newly added aromatic
                amino acids changed the original electron transfer chain to (F1p, Y27p+4, F24p+4) (electrons are
                transferred from position 1 of the first monomer to positions 27 and 24 of the fifth monomer, and
                finally to position 1 of the second monomer to complete the cycle), where the centroid distances between
                adjacent aromatic rings are (4.6 Å, 5.5 Å, 4.1 Å). In the mutant, according to our model prediction, the
                electron transfer chain would change to (F2p, Y27p+4, F24p+4). However, we can clearly observe that the
                centroid distance between the aromatic rings in the electron transfer chain of this mutant is
                significantly larger than that of the wild type, with distances of (7.1 Å, 5.5 Å, 9.5 Å). </p>
            <p class="cn d-none"><strong>在优化目标一中</strong>
                ，我们将野生型金属还原地杆菌菌毛氨基酸序列上的2号位的苏氨酸替换成苯丙氨酸。我们只选中菌毛结构中的由Dijkstra算法得到的最短电子传递路径（其中原有的芳香环用红色标记，突变体T2F新增的芳香环用黄色标记）可以得到以下结果。
            </p>
            <p class="cn d-none">
                我们可以发现由算法得到的电子传递链中存在若干缺口，这是由于提供缺口部分的芳香环的菌毛单体并不在我们的模型中。我们统计了原有的野生型菌毛和突变体菌毛内部的电子传递链和链上芳香族氨基酸的间距。我们可以发现新增的芳香族氨基酸将原有的电子传递链为（F1p，Y27p+4，F24p+4)（电子从第一个单体的1号位传递到第5个单体的27号位和24号位，最终传递给第二个单体的一号位实现循环）其中相邻芳香环的质心间距为（4.6埃，5.5埃，4.1埃）。而在突变体中，按照我们的模型预测的结果，电子传递链会变成（F2p，Y27p+4，F24p+4）。但我们可以明显的观察到，该突变体的电子传递链中芳香环质心间距较野生型明显变大，间距为（7.1埃，5.5埃，9.5埃）。
            </p>
            <p class="en">This situation arose because, when determining the constraint conditions of the decision
                variables in the optimization model, to simplify the model, we used the average coordinates of all
                carbon atoms on the side chain of the modified position to approximately predict the centroid
                coordinates of the newly added aromatic ring. The error caused by simplifying the model ultimately led
                to incorrect optimization results. </p>
            <p class="cn d-none">
                这种情况的出现是因为我们在确定优化模型中决策变量的约束条件的时候，为了简化模型，使用了所改动位置侧链上所有碳原子的平均坐标来近似预测新增的芳香环的质心坐标。简化模型导致的误差最终使得我们得到了错误的优化结果。
            </p>
            <p class="en">This model illustrates that objective function one can no longer be optimized by adding new
                aromatic amino acids to the original amino acid sequence. Subsequent teams may try to optimize by
                changing the type of aromatic amino acids to affect the overall hydrophilicity and hydrophobicity of the
                pilus.</p>
            <p class="cn d-none">
                这一模型可以很好的说明目标函数一已经无法通过在原有氨基酸的序列上添加新的芳香族氨基酸来进行优化。后续的团队可以尝试使用改变芳香族氨基酸的种类影响整体菌毛的亲疏水性这一思路来进行优化。</p>
            <div class="row">
                <img src="../asserts/img/model/model_10.jpg " alt="" class="col-md-12">
                <p>Fig.8 The comparison in electron transport chain between the WT（yellow）and the T2F（orange）</p>
                </img>
            </div>
            <p class="en"><strong>In objective function two</strong>we added 3, 4, and 5 phenylalanine residues at the
                positions calculated by our optimization algorithm, respectively, on top of the original structure. We
                can intuitively observe that the introduction of aromatic amino acids significantly improves the uniform
                distribution of aromatic amino acids in the original pili structure, causing the core of electron
                transfer to spread outward from the pili's central axis. This means that when the structure of
                conductive pili changes under the influence of different environmental factors, there are more aromatic
                rings involved in the electron transfer process, potentially leading to an increase in the overall
                electrical conductivity of the pili.</p>
            <p class="cn d-none"><strong>在优化目标二中</strong>
                我们分别在原有的结构基础上，在由我们的优化算法计算得到的位点上分别加入3，4，5个苯丙氨酸。我们可以非常直观的发现加入的芳香族氨基酸明显提高了原有菌毛结构中的芳香族氨基酸的分布的均匀度，使得电子传递的核心从菌毛的中轴开始向外扩散。这就意味着此时的导电菌毛在不同环境因素影响下结构发生变化时，菌毛中有更多的芳香环可能参与到电子传递的过程中，这就意味着菌毛的综合电导率可能会增加。
            </p>
            <div class="row">
                <img src="../asserts/img/model/model_11.jpg " alt="" class="col-md-12">
                <p>Fig 9.Optimization results of objective function 2</p>
                </img>
            </div>
            <h2 id="06" class="module">Semi-rational Directed Evolution</h2>
            <p class="en">To reduce the computational workload in modeling, a common practice is to simplify the model
                by introducing assumptions. For example, when determining constraints for optimizing, we approximate the
                centroid coordinates of the newly added aromatic ring by using the average coordinates of all atoms in
                the side chain at the modified position.However, introducing assumptions means that our model can not
                achieve the level of precision required in rational protein design. We can only identify several
                potential mutation sites, and we cannot guarantee that the final structure of the pili and the internal
                electron transfer will conform to our design.</p>
            <p class="cn d-none">
                为了减少建模的任务量，一个非常常见的做法就是通过引入假设来简化模型，例如我们在确定优化模型的约束条件时用侧链上所有原子的平均坐标来近似预测新增的芳香环的质心坐标。但引入假设就意味着我们的模型没有办法做到和蛋白质的理性设计中要求的精确程度。我们只能确定若干个可能的突变位点，并无法保证最终菌毛的结构以及内部的电子传递情况会符合我们的设计。
            </p>
            <p class="en">This situation is quite common in protein structure optimization, and the most common solution
                to address this is to use a non-rational (high-throughput screening) approach to compensate for the
                shortcomings of rational design.</p>
            <p class="cn d-none">这种情况在蛋白质的结构优化中是非常常见的，而应对这种情况的最常见的方案就是使用一种非理性（高通量的筛选）的方法来弥补掉理性设计上的不足。</p>
            <p class="en">Directed evolution simulates the process of Darwinian evolution in the laboratory by creating
                a large number of mutants through random mutation and recombination. Specific selection pressures are
                applied for desired features, allowing the selection of proteins with desired characteristics, achieving
                molecular-level simulated evolution.</p>
            <p class="cn d-none">定向进化是在实验室中模拟达尔文进化过程,通过随机突变和重组,人为制造大量的突变体,按照特定的需要和目的给予选择压力,筛选出具有期望特征的蛋白质,实现分子水平的模拟进化。”
            </p>
            <p class="en">Among the various methods of directed evolution, semi-rational directed evolution is the most
                widely applied with the most successful cases. This method aims to mutate proteins based on some
                understanding of their physicochemical properties, three-dimensional structure, structure-activity
                relationships, catalytic mechanisms, etc. With the assistance of computers, it then uses a reasonable
                high-throughput screening method to quickly obtain the target mutants.</p>
            <p class="cn d-none">
                而在定向进化的诸多方法中，半理性定向进化应用最广泛，成功案例最多。该方法旨在对蛋白的理化性质、三维结构、构效关系、催化机理等信息有一定理解的基础上，在计算机辅助下，对蛋白进行突变，并利用合理的高通量筛选方法，以快速获得目标突变体。
            </p>
            <p class="en">In our project, after identifying potential sites that may enhance the electrical conductivity
                of the pili, the next step is to construct a mutant library with these sites as hotspots and perform
                screening using the aforementioned methods. In our original plan, we intended to express and purify the
                mutants from the library one by one in engineered bacteria to measure their electrical conductivity.
                However, we quickly realized that this approach was too inefficient. Additionally, due to the relative
                conservation in the assembly of Type IV pili, a significant portion of mutant pili may fail to assemble
                correctly and lose their conductive function. Screening these pili would consume a considerable amount
                of effort.</p>
            <p class="cn d-none">
                在我们的项目中，在确定了可能提高菌毛的电导率的位点后，接下来我们需要以上述位点为突变热点构建突变体库，并使用一定的方法进行筛选。在我们原先的方案，我们准备将构建出的突变体库一个一个的在工程菌中表达并分离和纯化以测量其电导率。我们很快就发现这种方法的效率太低了，且由于四型菌毛组装的相对保守性，可能会有相当一部分的突变体菌毛是无法正确组装而失去导电功能的，对于这部分菌毛的筛选会浪费我们大量精力。
            </p>
            <p class="en">We plan to leverage the original function of e-pili, using Geobacter sulfurreducens as a
                chassis organism and combining it with the improved H-type microbial fuel cell model previously
                developed by previous researchers[8,9]. This bioelectrochemical system will be used for the initial
                screening of pili with higher electrical conductivity by observing the maximum current generated by the
                strains in microbial fuel cells.</p>
            <p class="cn d-none">我们准备从导电菌毛的本来的功能入手，以硫还原地杆菌作为底盘，结合研究者此前改进的H
                型燃料电池模型[8,9]，搭建生物电化学系统通过观察菌株在微生物燃料电池中的最大电流来初步筛选具有更高电导率的菌毛。
                电池中使用石墨电极，使用稳压器保证阳极电势，电池阳极和阴极之间连接电阻。细胞被接种到阳极之后，在含有10mM 的乙酸钠（电子供体）和 40mM 延胡索酸钠（电子受体）的 freshwater培养基中生长。
            </p>
            <div class="row">
                <div class="col-md-1"></div>
                <img src="../asserts/img/model/model_12.jpg " alt="" class="col-md-10">
                <p>Fig10.The Microbial Fuel Cell Model</p>
                </img>
            </div>
            <h2 id="07" class="module">Future work</h2>
            <p class="en">To validate our design, we still need to measure the electrical conductivity of individual
                pili. However, due to project timelines and the unavailability of the necessary equipment (low-noise
                nanoelectrode measurement platform and atomic force microscope [10]), it is regrettable that we won't be
                able to experimentally verify our modeling results this year.</p>
            <p class="cn d-none">
                要验证我们的设计，我们最终仍然需要测量单根菌毛的电导率，然而受限于项目的周期以及我们无法获得相关的仪器设备（低噪声纳米电极测量平台和原子力显微镜[10]），非常遗憾的是今年我们无法在湿实验中验证我们的建模结果。
            </p>
            <p class="en">We plan to continue our research after the competition ends, and we sincerely hope that future
                iDEC teams can refer to the construction approach of our model to design proteins that meet their
                project requirements.</p>
            <p class="cn d-none">我们将在比赛结束后继续我们的研究，我们也衷心的希望未来的iDEC团队能够参考我们的模型的构建思路来设计符合他们的项目需求的蛋白质。</p>
            <div class="module" id="08" style="padding-top: 2rem;"></div>
            <div class="accordion-container">
                <button class="accordion">References</button>
                <div class="panel">
                    <p>[1] Tan, Y., Adhikari, R.Y., Malvankar, N.S., Pi, S., Ward, J.E., Woodard, T.L., Nevin, K.P.,
                        Xia, Q., Tuominen, M.T. and Lovley, D.R. (2016), Synthetic Biological Protein Nanowires with
                        High Conductivity. Small, 12: 4481-4485. </p>
                    <p>[2]Nikhil S. Malvankar , Madeline Vargas , Kelly Nevin , Pier-Luc Tremblay , Kenneth
                        Evans-Lutterodt , Dmytro Nykypanchuk , Eric Martz , Mark T. Tuominen , and Derek R. Lovley
                        Structural Basis for Metallic-Like Conductivity in Microbial Nanowires,10.1128/mbio.00084-15</p>
                    <p>[3] Toshiyuki Ueki, David J.F. Walker, Pier-Luc Tremblay, Kelly P. Nevin, Joy E. Ward, Trevor L.
                        Woodard, Stephen S. Nonnenmann, and Derek R. Lovley ACS Decorating the Outer Surface of
                        Microbially Produced Protein Nanowires with PeptidesSynthetic Biology 2019 8 (8), 1809-1817
                        DOI:10.1021/acssynbio.9b00131</p>
                    <p>[4] Tan Y, Adhikari RY, Malvankar NS, Ward JE, Woodard TL, Nevin KP, Lovley DR. 2017. Expressing
                        the Geobacter metallireducens PilA in Geobacter sulfurreducens yields pili with exceptional
                        conductivity. mBio 8:e02203-16.</p>
                    <p>[5]Walker DJF, Adhikari RY, Holmes DE, Ward JE, Woodard TL, Nevin KP, Lovley DR. 2018.
                        Electrically conductive pili from genes of phylogenetically diverse microorganisms. ISME J
                        12:48-58.</p>
                    <p>[6]Yang Tan , Ramesh Y. Adhikari , Nikhil S. Malvankar , Joy E. Ward , Trevor L. Woodard , Kelly
                        P. Nevin , and Derek R. Lovley Expressing the Geobacter metallireducens PilA in Geobacter
                        sulfurreducens Yields Pili with Exceptional Conductivity.</p>
                    <p>[7]Campos M, Cisneros D A, Nivaskumar M, et al. The type II secretion system - a dynamic fiber
                        assembly nanomachine[J]. Res Microbiol. 2013, 164(6): 545-555</p>
                    <p>[8]Nevin, K. P. et al. Anode biofilm transcriptomics reveals outer surface components essential
                        for high density current production in Geobacter sulfurreducens fuel cells. PLoS One 4, e5628
                        (2009).</p>
                    <p>[9]Bond, D. R. & Lovley, D. R. Electricity Production by Geobacter sulfurreducens Attached to
                        Electrodes. Appl. Environ. Microbiol. 69, 1548–1555 (2003).</p>
                    <p>[10] Ramesh Y. Adhikari ,a Nikhil S. Malvankar,‡ab Mark T. Tuominena and Derek R. Lovley*b
                        Conductivity of individual Geobacter pili 10.1039/C5RA28092C</p>
                </div>
            </div>
    </article>
    <!-- main page content -->
    </main>
    <!-- JS -->
    <script src="../asserts/js/jquery-3.7.0.js"></script>
    <script src="../asserts/js/bootstrap.bundle.min.js"></script>
    <script src="../asserts/js/my.js"></script>
    <script src="../asserts/js/mathjax.js"></script>
</body>

</html>